sdk/servicebus/service-bus/src/constructorHelpers.ts

```diff
@@ -24,6 +25,12 @@ export interface ServiceBusClientOptions {
    * Options to configure the channelling of the AMQP connection over Web Sockets.
    */
   webSocketOptions?: WebSocketOptions;
+  /**
+   * Options for adding user agent details to outgoing requests.
+   *
+   * @type {UserAgentOptions}
```

Comment:
Nit: I don't think this line adds any value, doc tools that read TypeScript already know the type of this field (would be really interested if this was not the case.)

Reply:
Well, I've removed it. 
This `@type` info is added when you use `Document This` extension on VSCode and other packages have it too.